IdentityCore/src/MSIDConstants.m

```diff
@@ -39,3 +53,17 @@
 NSString *const MSIDTrustedAuthorityWorldWide    = @"login.microsoftonline.com";
 NSString *const MSIDTrustedAuthorityUSGovernment = @"login-us.microsoftonline.com";
 NSString *const MSIDTrustedAuthorityCloudGovApi  = @"login.usgovcloudapi.net";
+
+/**
+ * @discussion Workaround for exporting symbols from category object files. See article
+ * https://medium.com/ios-os-x-development/categories-in-static-libraries-78e41f8ddb96#.aedfl1kl0
+ */
+__attribute__((used)) static void importCategories() {
```

Comment:
Can we use -ObjC flag on the app side instead?
This solution seems error prone, seems as we add new categories, we would need to also add symbols here

Reply:
I already tried `-ObjC` flag but it didn't work. That's why I added this. In addition to that I tried to avoid additional steps on app side in terms of user experience because it could avoid a common mistake for customers who don't really read documentations.

Reply:
The less flags the customers has to put the less error prone it is for them. It's true that we have to keep it in mind while developing the lib but if we test it in an application it just won't compile and remind us about it.

Reply:
I'm happy to explore other options, but I feel that current solution is pretty hacky. We have been following same -ObjC path for ADAL Obj-C and never run into issues with that. Having said that, if the desire is to limit number of steps developer needs to do, there're flags we can introduce on the library side as well to prevent those kind of issues. 

All over, I feel that this solution is pretty hacky and will be difficult to maintain for new developers. Happy to sync offline and try finding a better solution.

Reply:
I assume you've been building dynamic framework. This is static library and that's why we are seeing the issue. Let's sync offline.

Reply:
We couldn't use `-ObjC` in customers' applications due to integration behavior changes. I've talked about it in the team but we couldn't ask this to our customers not to break our SDK's one-liner integration.

Reply:
What's about GENERATE_MASTER_OBJECT_FILE?